src/user/user.spec.js

```diff
@@ -54,4 +54,22 @@ describe("user :", () => {
       await expect(xo.createUser(simpleUser)).rejects.toMatchSnapshot();
     });
   });
+
+  describe("delete", () => {
+    it("a user successfully with id", async () => {
```

Comment:
This results in correct sentences in the snapshots, but in the test it's not consistent.
I can do it another way :
```
describe ( "delete()", () => {
    it("deletes a user successfully with id", async () => {
        ...
    });
});
```
In this case, I have to change the titles in the "user.create" too.
What is the best thing to do ?

Reply:
The second proposition is better.